src/zenml/zen_stores/schemas/user_schemas.py

```diff
@@ -129,6 +129,7 @@ def to_model(
                 full_name=self.full_name,
                 created=self.created,
                 updated=self.updated,
+                roles=[ra.role.to_model(_block_recursion=True) for ra in self.assigned_roles]
```

Comment:
Can you explain why `roles` get set if blocking recursion, but e.g. teams are not?

Reply:
removed this inconsistency